src/lib/api/work.fragments.js

```diff
@@ -782,6 +782,26 @@ export function pidToWorkId({ pid }) {
   };
 }
 
+export function faustToWork({ faust }) {
```

Comment:
Du kan kalde den her for `faustToWorkId`, så den passer sammen med `oclcToWorkId` og `pidToWorkId`

Reply:
det er jo faktisk de to andre som har misvisende navne :)